pkg/controller/volume/attachdetach/reconciler/reconciler.go

```diff
@@ -367,3 +385,17 @@ func (rc *reconciler) reportMultiAttachError(volumeToAttach cache.VolumeToAttach
 	// Log all pods for system admin
 	klog.InfoS("Multi-Attach error: volume is already used by pods", "pods", klog.KObjs(pods), "attachedTo", otherNodesStr, "volume", volumeToAttach)
 }
+
+// hasOutOfServiceTaint returns true if the node has out of service taint present
+// with `NoExecute` effect.
+func (rc *reconciler) hasOutOfServiceTaint(nodeName types.NodeName) (bool, error) {
```

Comment:
It looks cool to move it to the previous position of reconcile().

Reply:
I am trying to understand the reasoning behind these placements. Do we have any such standards on placing the functions? 
By the way, I have not any strong opinions here and am fine moving it. Just that I added it to below to keep the diff look simpler.

Reply:
Current code has small utility functions before reconcile(), so it looks natural to insert new one like them, IMO. You can ignore my comment.

Reply:
Sure. I will address this. Thank you